pandas/io/parquet.py

```diff
@@ -296,6 +328,11 @@ def read_parquet(path, engine: str = "auto", columns=None, **kwargs):
         If not None, only these columns will be read from the file.
 
         .. versionadded:: 0.21.1
+    use_nullable_dtypes : bool, default False
+        If True, use dtypes that use ``pd.NA`` as missing value indicator
+        for the resulting DataFrame (only applicable for ``engine="pyarrow"``).
+        As new dtypes are added that support ``pd.NA`` in the future, the
+        output with this option will change to use those dtypes.
```

Comment:
versionadded

also should this option be marked as experimental so that it can be freely changed between now and 2.0 is desired. also should using this keyword issue a FutureWarning

Reply:
pls add a versionadded

Reply:
I added a versionadded and a note that this keyword is experimental. I would personally leave out the future warning, as that gets annoying (but for sure much more explicit about it being experimental).